src/app/calendar/page.tsx

```diff
@@ -1,11 +1,24 @@
 import { Metadata } from "next";
 import React from "react";
 import ParcelCalendar from "@/app/calendar/ParcelCalendar";
-import { getParcelsWithCollectionDate } from "@/app/calendar/parcelCalendarFunctions";
+import supabase from "@/supabaseServer";
+import { ParcelsWithExtraFields } from "@/app/calendar/parcelCalendarFunctions";
+
+export const dynamic = "force-dynamic";
+
+const getParcelsWithCollectionDate = async (): Promise<ParcelsWithExtraFields[]> => {
+    const { data, error } = await supabase
+        .from("parcels")
+        .select("*, clients ( full_name ), collection_centres (name)")
+        .not("collection_datetime", "is", null);
+    if (error) {
+        throw new Error("Database error");
```

Comment:
It is a VFB-22 thing but should we make the error message consistent?

Reply:
Yes, think I change it already in my branch tho